projects/fsastorefrontlib/src/cms-components/fnol/fnol-navigation/fnol-navigation.component.html

```diff
@@ -13,7 +13,12 @@
         class="primary-button btn-block"
         (click)="next(activeStepIndex, claimRequest)"
       >
-        {{ 'common.continue' | cxTranslate }}
+        {{
+          (activeStepData.name === 'Summary'
```

Comment:
Maybe we should check if it is "activeStepData.summaryStep" instead of checking step name